radio/src/targets/common/arm/stm32/usb_driver.cpp

```diff
@@ -116,7 +116,6 @@ bool usbStarted()
   return usbDriverStarted;
 }
 
-#if defined(USB_JOYSTICK)
```

Comment:
Sure, we want this to be in the bootloader?

Reply:
Ah didn't think of that. Nope, saved space is always welcome. With the right condition though :P